webcompat/static/css/development/components/style-variable.css

```diff
@@ -0,0 +1,57 @@
+/*----------Variable----------*/
+
+/* base */
+:root {
```

Comment:
Oh, here they are. So these are "base" variables? And the ones in each module/file are "local" (or something)?